wiki/Glossary/Elite_Beat_Agents/en.md

```diff
@@ -7,9 +7,9 @@ tags:
 
 # Elite Beat Agents
 
-**Elite Beat Agents** or *EBA* refers to the game called [Elite Beat Agents](https://en.wikipedia.org/wiki/Elite_Beat_Agents) developed by [iNiS](https://en.wikipedia.org/wiki/INiS) and released for the [Nintendo DS](https://en.wikipedia.org/wiki/Nintendo_DS) in 2006.
+*See also: [osu! Tatake! Ouendan](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan "Wikipedia")* <!-- this needs an stub to link to -->
```

Comment:
```suggestion
*See also: [Osu! Tatake! Ouendan](https://en.wikipedia.org/wiki/Osu!_Tatakae!_Ouendan "Wikipedia")* <!-- this needs an stub to link to -->
```
The ouendan games capitalize the "o" in "Osu!." Whereas our osu! likes to be special and spells it with a lowercase o